packages/framework/src/parser.ts

```diff
@@ -266,7 +269,7 @@ export const EnsembleParser = {
       | undefined;
     return {
       id: get(menu, [menuType, "id"]) as string | undefined,
-      type: String(menuType),
+      type: menuType.toString(),
```

Comment:
You need to either add optional chaining `?` to prevent an exception. Using the string constructor was for this purpose also.